src/Apps/Artist/routes.tsx

```diff
@@ -36,7 +37,7 @@ import { ShowProps } from "./Routes/Shows"
 // * `render` functions requires casting
 // * `Redirect` needs to be casted, as it’s not compatible with `RouteConfig`
 export const routes: RouteConfig[] = [
-  {
+  new Route({
```

Comment:
Trying to think of how to _require_ the usage of this class 🤔

Reply:
Is this totally required now? I know Taion mentioned in Slack that they wrapped their routes with a custom route to handle cases like this but I feel like we could find a way to hoist this up into some kind of factory function at the `buildClientApp` or `buildServerApp` level so that we're not introducing scope complexity and deviating from Found docs. 

Ideally we can keep this as close to Found as possible, or PR upstream, or find a way to hide this complexity at the framework level vs the `routes` definition.

Reply:
What they [Taion] do is use JSX to define the routes, which also alleviates this issue and doesn't deviate from the Found docs. (It also replaces the word `children` for JSX nesting, which I personally find a little more intuitive, but not sure if that applies to others as well.)

I'm on the fence about either, if you feel strongly about staying with object literals and doing recursive work on each request, then I'll change it to do that.

Reply:
I do feel (slightly) strongly about it just based on previous experience working on big react router apps where JSX feels slighly more natural until the route tree gets to a certain size  and enough JS is interpolated inside of JSX props that an array of objects feels nicer, and in any case it's native JS so devs are already familiar. When `makeRouteConfig` takes in a JSX tree it's parsed and turned back into this anyway, and by us doing this behind the scenes it requires one less import on the devs side.

I guess what I'm saying is that JSX can quickly get unweildy for little gain and unless we're plan on nesting children 1000 levels deep I don't see any real perf implications.

Reply:
Got it, will update accordingly then 👍